INFO:root:Openning session
DEBUG:pyvisa:SerialSession was correctly imported.
DEBUG:pyvisa:USBSession and USBRawSession were not imported No backend available.
DEBUG:pyvisa:TCPIPSession was correctly imported.
DEBUG:pyvisa:GPIBSession was not imported 'MockGPIB' object is not subscriptable.
DEBUG:pyvisa:Created library wrapper for py
DEBUG:pyvisa:Created ResourceManager with session 7227428
INFO:root:Configuring meter
DEBUG:root:Sending command :ACQU:STAR to meter
WARNING:root:Command :ACQU:STAR cannot be written
Traceback (most recent call last):
  File "C:\Users\manue\OneDrive\Documentos\Proyecto CATION\Pruebas CITIUS (Array antiguo)\interrogador_v0\test_socket.py", line 125, in write
    self.inst_handler.write(cmd)
    ^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'ResourceManager' object has no attribute 'write'
DEBUG:root:Sending command :ACQU:CONF:THRE:CHAN:0:8 to meter
WARNING:root:Command :ACQU:CONF:THRE:CHAN:0:8 cannot be written
Traceback (most recent call last):
  File "C:\Users\manue\OneDrive\Documentos\Proyecto CATION\Pruebas CITIUS (Array antiguo)\interrogador_v0\test_socket.py", line 125, in write
    self.inst_handler.write(cmd)
    ^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'ResourceManager' object has no attribute 'write'
DEBUG:root:Sending command :ACQU:CONF:RATE:1 to meter
WARNING:root:Command :ACQU:CONF:RATE:1 cannot be written
Traceback (most recent call last):
  File "C:\Users\manue\OneDrive\Documentos\Proyecto CATION\Pruebas CITIUS (Array antiguo)\interrogador_v0\test_socket.py", line 125, in write
    self.inst_handler.write(cmd)
    ^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'ResourceManager' object has no attribute 'write'
DEBUG:root:Sending command :ACQU:CONF:RANG:ENAB to meter
WARNING:root:Command :ACQU:CONF:RANG:ENAB cannot be written
Traceback (most recent call last):
  File "C:\Users\manue\OneDrive\Documentos\Proyecto CATION\Pruebas CITIUS (Array antiguo)\interrogador_v0\test_socket.py", line 125, in write
    self.inst_handler.write(cmd)
    ^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'ResourceManager' object has no attribute 'write'
DEBUG:root:Sending command :ACQU:CONF:RANG:WAVE:0:5:1518.000, 1528.000, 1528.100, 1538.000, 1538.100, 1548.000, 1548.100, 1558.000, 1558.100, 1568.000 to meter
WARNING:root:Command :ACQU:CONF:RANG:WAVE:0:5:1518.000, 1528.000, 1528.100, 1538.000, 1538.100, 1548.000, 1548.100, 1558.000, 1558.100, 1568.000 cannot be written
Traceback (most recent call last):
  File "C:\Users\manue\OneDrive\Documentos\Proyecto CATION\Pruebas CITIUS (Array antiguo)\interrogador_v0\test_socket.py", line 125, in write
    self.inst_handler.write(cmd)
    ^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'ResourceManager' object has no attribute 'write'
DEBUG:root:Sending command :ACQU:CONF:RANG:FOR:0: to meter
WARNING:root:Command :ACQU:CONF:RANG:FOR:0: cannot be written
Traceback (most recent call last):
  File "C:\Users\manue\OneDrive\Documentos\Proyecto CATION\Pruebas CITIUS (Array antiguo)\interrogador_v0\test_socket.py", line 125, in write
    self.inst_handler.write(cmd)
    ^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'ResourceManager' object has no attribute 'write'
DEBUG:root:Sending command :ACQU:STOR to meter
WARNING:root:Command :ACQU:STOR cannot be written
Traceback (most recent call last):
  File "C:\Users\manue\OneDrive\Documentos\Proyecto CATION\Pruebas CITIUS (Array antiguo)\interrogador_v0\test_socket.py", line 125, in write
    self.inst_handler.write(cmd)
    ^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'ResourceManager' object has no attribute 'write'
DEBUG:root:Sending command :ACQU:STOP to meter
WARNING:root:Command :ACQU:STOP cannot be written
Traceback (most recent call last):
  File "C:\Users\manue\OneDrive\Documentos\Proyecto CATION\Pruebas CITIUS (Array antiguo)\interrogador_v0\test_socket.py", line 125, in write
    self.inst_handler.write(cmd)
    ^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'ResourceManager' object has no attribute 'write'
INFO:root:Checking configuration ... 
WARNING:root:Exception occurred during querying command IDEN?
Traceback (most recent call last):
  File "C:\Users\manue\OneDrive\Documentos\Proyecto CATION\Pruebas CITIUS (Array antiguo)\interrogador_v0\test_socket.py", line 135, in query
    response = self.inst_handler.query(cmd)
               ^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'ResourceManager' object has no attribute 'query'
DEBUG:root:Command: IDEN?
DEBUG:root:Response: None
WARNING:root:Exception occurred during querying command :STAT?
Traceback (most recent call last):
  File "C:\Users\manue\OneDrive\Documentos\Proyecto CATION\Pruebas CITIUS (Array antiguo)\interrogador_v0\test_socket.py", line 135, in query
    response = self.inst_handler.query(cmd)
               ^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'ResourceManager' object has no attribute 'query'
DEBUG:root:Command: :STAT?
DEBUG:root:Response: None
WARNING:root:Exception occurred during querying command :ACQU:STAR
Traceback (most recent call last):
  File "C:\Users\manue\OneDrive\Documentos\Proyecto CATION\Pruebas CITIUS (Array antiguo)\interrogador_v0\test_socket.py", line 135, in query
    response = self.inst_handler.query(cmd)
               ^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'ResourceManager' object has no attribute 'query'
DEBUG:root:Command: :ACQU:STAR
DEBUG:root:Response: None
WARNING:root:Exception occurred during querying command :RECA
Traceback (most recent call last):
  File "C:\Users\manue\OneDrive\Documentos\Proyecto CATION\Pruebas CITIUS (Array antiguo)\interrogador_v0\test_socket.py", line 135, in query
    response = self.inst_handler.query(cmd)
               ^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'ResourceManager' object has no attribute 'query'
DEBUG:root:Command: :RECA
DEBUG:root:Response: None
WARNING:root:Exception occurred during querying command :ACQU:CONF:THRE:CHAN:0?
Traceback (most recent call last):
  File "C:\Users\manue\OneDrive\Documentos\Proyecto CATION\Pruebas CITIUS (Array antiguo)\interrogador_v0\test_socket.py", line 135, in query
    response = self.inst_handler.query(cmd)
               ^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'ResourceManager' object has no attribute 'query'
DEBUG:root:Command: :ACQU:CONF:THRE:CHAN:0?
DEBUG:root:Response: None
WARNING:root:Exception occurred during querying command :ACQU:CONF:RANG:WAVE:0?:ACQU:CONF:RANG:FORM:0?:ACQU:CONF:RANG:STAT:0?:ACQU:STOP
Traceback (most recent call last):
  File "C:\Users\manue\OneDrive\Documentos\Proyecto CATION\Pruebas CITIUS (Array antiguo)\interrogador_v0\test_socket.py", line 135, in query
    response = self.inst_handler.query(cmd)
               ^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'ResourceManager' object has no attribute 'query'
DEBUG:root:Command: :ACQU:CONF:RANG:WAVE:0?:ACQU:CONF:RANG:FORM:0?:ACQU:CONF:RANG:STAT:0?:ACQU:STOP
DEBUG:root:Response: None
WARNING:root:Exception occurred during querying command :STAT?
Traceback (most recent call last):
  File "C:\Users\manue\OneDrive\Documentos\Proyecto CATION\Pruebas CITIUS (Array antiguo)\interrogador_v0\test_socket.py", line 135, in query
    response = self.inst_handler.query(cmd)
               ^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'ResourceManager' object has no attribute 'query'
INFO:root:... None
DEBUG:root:Sending command :ACQU:STAR to meter
WARNING:root:Command :ACQU:STAR cannot be written
Traceback (most recent call last):
  File "C:\Users\manue\OneDrive\Documentos\Proyecto CATION\Pruebas CITIUS (Array antiguo)\interrogador_v0\test_socket.py", line 125, in write
    self.inst_handler.write(cmd)
    ^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'ResourceManager' object has no attribute 'write'
DEBUG:root:Sending command :ACQU:RECA to meter
WARNING:root:Command :ACQU:RECA cannot be written
Traceback (most recent call last):
  File "C:\Users\manue\OneDrive\Documentos\Proyecto CATION\Pruebas CITIUS (Array antiguo)\interrogador_v0\test_socket.py", line 125, in write
    self.inst_handler.write(cmd)
    ^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'ResourceManager' object has no attribute 'write'
WARNING:root:Exception occurred during querying command :ACQU:OSAT:CHAN:0?
Traceback (most recent call last):
  File "C:\Users\manue\OneDrive\Documentos\Proyecto CATION\Pruebas CITIUS (Array antiguo)\interrogador_v0\test_socket.py", line 135, in query
    response = self.inst_handler.query(cmd)
               ^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'ResourceManager' object has no attribute 'query'
DEBUG:pyvisa:Closing ResourceManager (session: 7227428)
